spec/unit/source/git_spec.rb

```diff
@@ -139,6 +139,36 @@
     end
   end
 end
+#
+# describe R10K::Source::Git, 'when prefixing is enabled, and the prefix name is overridden' do
```

Comment:
These tests need to be implemented or deleted, committing in commented out code doesn't provide any value.